cmdstanpy/utils.py

```diff
@@ -390,7 +390,7 @@ def rdump(path: str, data: Dict) -> None:
             print(line, file=fd)
 
 
-def rload(fname: str) -> dict:
+def rload(fname: str) -> Dict:
```

Comment:
```suggestion
def rload(fname: str) -> dict:
```

I think py39 recommends using base items and not typing items, but really I don't care.

Reply:
we need to be consistent - elsewhere found type hint as `Dict`.
whatever is best practice is fine by me.

found this:  https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html
```
# In Python 3.8 and earlier, the name of the collection type is
# capitalized, and the type is imported from 'typing'
x: List[int] = [1]
```

Reply:
Then it is ok